elementary/clients/slack/client.py

```diff
@@ -112,6 +118,14 @@ def send_report(self, channel: str, report_file_path: str):
             logger.error("Failed to send report to Slack.")
         return send_succeed
 
+    def get_user_id_from_email(self, email: str) -> str:
+        logger.info(f"Attempting to get slack id of user: {email}")
+        try:
+            return self.client.users_lookupByEmail(email=email)["user"]["id"]
```

Comment:
What happens when the email is not found? Is the response None or will it cause a `SlackApiError`?

Reply:
It returns None unless there is an error with the call to the SlackAPI - a missing email just returns the original handle with the latest changes.